src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -24,9 +24,11 @@ export const AllEarnOpportunities = () => {
       </Card.Header>
       <Card.Body pt={0}>
         <Stack spacing={2} mt={2} mx={-4}>
-          {vaults.map(vault => (
-            <EarnOpportunityRow {...vault} key={vault.vaultAddress} isLoaded={!!vault} />
-          ))}
+          {vaults
```

Comment:
cool. it would be nice to memo this and only pass in an id here to avoid rerenders, but that's outside the scope of this PR

Reply:
done!